### ISSUE TITLE
TableEditor select rows with same value

### ISSUE THREAD

**Issue by ebocher** [orbisgis/orbisgis#933]
The action "Select rows with the same value" returns the message

```java
"Algo error"
```

The SQL query seems correct. The problem is related to the ReadRowSetImpl and the method private void fetchBatchPk(int targetBatch)

**Comment by nicolas-f** [comment]
Thursday or Friday

**Comment by nicolas-f** [comment]
Working on it

**Comment by nicolas-f** [comment]
Fixed

**Comment by nicolas-f** [comment]
tested, fixed

**Comment by ebocher** [comment]
Doesn't work with a postgis connection.
It return this message

```java
Find field value with the following request:
SELECT "pk" FROM route WHERE  "nature" = 'Autoroute'
Statement a été fermé.
```

**Comment by ebocher** [comment]
@gpetit 
Could you confirm ?

**Comment by ebocher** [comment]
Must be fixed on 5.1 before a new release

**Comment by ebocher** [comment]
Fixed for 5.1.
